src/ric/319C4.c

```diff
@@ -2148,7 +2151,7 @@ void EntityGiantSpinningCross(Entity* self) {
         self->ext.giantcross.unk7E = 0x400;
         g_api.PlaySfx(0x661);
         self->step++;
-        primUVCoords = &D_8015635C[0][0];
+        primUVCoords = &D_8015635C[0];
```

Comment:
This should be assignable without a reference to the first element - `primUVCoords = D_8015635C`.

Reply:
It's good to have an indicator that this is an array at first glance